src/components/ChartJsWrapper/index.jsx

```diff
@@ -33,31 +29,29 @@ const ChartJsWrapper = ({
   type,
   chartHeight,
   spinnerSize,
-  missingDataError = false,
 }) => {
-  let showError;
-
-  if (data && missingDataError) {
-    showError = data.datasets.every(dataset => {
-      const latestDataDate = new Date(dataset.data[dataset.data.length - 1].x);
-      const currentDate = new Date(); // get current date
-      const timeDifference = Math.abs(
-        currentDate.getTime() - latestDataDate.getTime()
-      );
-      const daysDifference = Math.ceil(timeDifference / (1000 * 3600 * 24));
+  if (data) {
+    if (
```

Comment:
Yes, this was made complicated so it would snot complain if at least one series has recent data.  Your suggestion of showing the chart, with the error can simplify this logic.